scipy/stats/tests/test_stats.py

```diff
@@ -6010,7 +6010,11 @@ def test_describe_empty(self):
         assert_raises(ValueError, stats.describe, [])
 
 
-def test_normalitytests():
+@pytest.mark.skip_xp_backends(cpu_only=True,
+                              reasons=['Uses NumPy for pvalue'])
+@pytest.mark.usefixtures("skip_xp_backends")
+@array_api_compatible
+def test_normalitytests(xp):
```

Comment:
Currently, the tests for the three functions are mixed, which means that NumPy tests are running multiple times next to alternative backend tests. That's fine for now, since the other functions will get converted soon.

One thing I think I'd like to do going forward - although I've done it it some of the first PRs like this - is to try to leave the test with the default dtype (unless the purpose is to test behavior for different dtypes). It will reduce the diff, but I think it's also more representative of what users will get in practice; that is, the default dtype is probably the most important to check.

Could you convert `st_skew`, `pv_skew` and `x` into `xp`-arrays at the beginning with default dtype and avoid all the repeated conversions?